### PR TITLE
Add stdio-mgr recipe to conda-forge

### PR DESCRIPTION
**Pull request by anjos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by anjos** [comment]
@conda-forge/help-python ping

**Comment by ocefpaf** [comment]
> * GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

@bskinn can you comment here saying you are OK being a maintainer for this feedstock?

**Comment by bskinn** [comment]
Confirmed, ok here.

I may need some hand holding, I'm new to conda-forge recipe maintenance, but I'm interested to learn.

File: recipes/stdio-mgr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - attrs >=17.1
```
Comment (ocefpaf) [inline recipes/stdio-mgr/meta.yaml:21]: This is probably not needed at build time, unless the package imports itself. Can you try to remove it and see how it goes?

```suggestion
```

Comment (bskinn) [reply recipes/stdio-mgr/meta.yaml:21]: Agreed, attrs should not be needed at build.

Comment (bskinn) [reply recipes/stdio-mgr/meta.yaml:21]: Ah, nope, was thinking this was a different package.

v1.0.1 here *does* import the package at build, to retrieve the version number. One of many overdue points of needed maintenance.

Comment (anjos) [reply recipes/stdio-mgr/meta.yaml:21]: Sure - I overlooked this.  Let me push a fix.

**Comment by anjos** [comment]
@bskinn: without the `attrs` package on the host, it does not build. I'll add it back.  See [logs here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=617968&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823).

**Comment by bskinn** [comment]
@anjos -- yeah, I need to update my setup.py here so that it extracts the version number without importing the whole package. One of many overdue points of maintenance.

**Comment by anjos** [comment]
No worries, @bskinn - we can iterate on this more easily now.

**Pull request merged**